src/Features/Core/Portable/PopulateSwitch/PopulateSwitchExpressionHelpers.cs

```diff
@@ -17,7 +17,7 @@ internal static class PopulateSwitchExpressionHelpers
         public static ICollection<ISymbol> GetMissingEnumMembers(ISwitchExpressionOperation operation)
         {
             var switchExpression = operation.Value;
-            var switchExpressionType = switchExpression?.Type;
+            var switchExpressionType = (INamedTypeSymbol?)switchExpression?.Type;
```

Comment:
how do you know it's an INamedType here?  if verified earlier, can you add comment?

Reply:
I've refactored the code so we only convert to an INamedTypeSymbol if the type is a named type.